packages/gatsby-telemetry/src/error-helpers.js

```diff
@@ -1,6 +1,14 @@
 const StackUtils = require(`stack-utils`)
 
-module.exports = tags => {
+const cleanPaths = str => {
+  const localPathRegex = new RegExp(
+    process.cwd().replace(/[-[/{}()*+?.\\^$|]/g, `\\$&`),
+    `g`
+  )
+  return str.replace(localPathRegex, `$PWD`)
+}
+
+const sanitizeError = tags => {
   if (!tags) return
   const { error } = tags
   if (error) {
```

Comment:
with the above change you can drop this line
```suggestion
  if (error) {
```

Reply:
Done